src/components/long-number.tsx

```diff
@@ -9,7 +9,8 @@ type Props = {
   children: string | number;
 };
 
-const LongNumber: FC<Props> = ({ children }: Props) =>
-  formatLargeNumber(children);
+const LongNumber: FC<Props> = ({ children }) => (
+  <>{formatLargeNumber(children)}</>
```

Comment:
Had to use a fragment to make TSC happy

Reply:
In that case were you are specifying the type of `children` and explicitely returning a Fragment you might be able to leave out the FC annotation and just type this as a standard function (`const LongNumber = ({ children }: Props) => (`)

Reply:
Updated